doc/source/ray-air/examples/gptj_inference.ipynb

```diff
@@ -212,6 +219,146 @@
+    "Same as in the batch inference workload, if you want to use inter-node model parallelism, you can also increase `num_gpus`. As we have created the model with `device_map=\"auto\"`, it will be automatically placed on correct devices. Note that this requires nodes with multiple GPUs.\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ray import serve\n",
+    "from starlette.requests import Request\n",
+    "\n",
+    "\n",
+    "@serve.deployment(ray_actor_options={\"num_gpus\": 1})\n",
+    "class PredictDeployment:\n",
+    "    def __init__(self, model_id: str, revision: str = None):\n",
+    "        from transformers import AutoModelForCausalLM, AutoTokenizer\n",
+    "        import torch\n",
+    "\n",
+    "        self.model = AutoModelForCausalLM.from_pretrained(\n",
+    "            model_id,\n",
+    "            revision=revision,\n",
+    "            torch_dtype=torch.float16,\n",
+    "            low_cpu_mem_usage=True,\n",
+    "            device_map=\"auto\",  # automatically makes use of all GPUs available to the Actor\n",
+    "        )\n",
+    "        self.tokenizer = AutoTokenizer.from_pretrained(model_id)\n",
+    "\n",
+    "    def generate(self, text: str) -> pd.DataFrame:\n",
+    "        input_ids = self.tokenizer(text, return_tensors=\"pt\").input_ids.to(\n",
+    "            self.model.device\n",
+    "        )\n",
+    "\n",
+    "        gen_tokens = self.model.generate(\n",
+    "            input_ids,\n",
+    "            do_sample=True,\n",
+    "            temperature=0.9,\n",
+    "            max_length=100,\n",
+    "        )\n",
+    "        return pd.DataFrame(\n",
+    "            self.tokenizer.batch_decode(gen_tokens), columns=[\"responses\"]\n",
+    "        )\n",
+    "\n",
+    "    async def __call__(self, http_request: Request) -> str:\n",
+    "        json_request: str = await http_request.json()\n",
+    "        prompts = []\n",
+    "        for prompt in json_request:\n",
+    "            text = prompt[\"text\"]\n",
+    "            if isinstance(text, list):\n",
+    "                prompts.extend(text)\n",
+    "            else:\n",
+    "                prompts.append(text)\n",
+    "        return self.generate(prompts)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can now `bind` the deployment with our arguments, and use {meth}`~ray.serve.run` to start it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "RayServeSyncHandle(deployment='PredictDeployment')"
+      ]
+     },
+     "execution_count": 18,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "deployment = PredictDeployment.bind(model_id=model_id, revision=revision)\n",
+    "serve.run(deployment)"
```

Comment:
Can we also add a note on how to use the "serve run file:deployment" syntax? We can link to the serve docs as well here.